Costume/052_Meowth.yaml

```diff
@@ -0,0 +1,21 @@
+species: Meowth
+imagename: Litten
+displayname: Litten
+setname: Costume
+tags: [costume]
+item: [Figy Berry]
+ability: [Blaze]
+moves:
+    - [Lick]
+    - [Fire Fang]
+    - [Roar]
+    - [Scary Face]
```

Comment:
Swap lick and fire fang order. Also I haven't run calcs but this set sounds very weak for a meowth. even for a LC meta